packages/amazonq/src/auth/util.ts

```diff
@@ -0,0 +1,61 @@
+/*!
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { AuthUtil } from 'aws-core-vscode/codewhisperer'
+import { AuthStatus } from '../../../core/dist/src/shared/telemetry/telemetry'
```

Comment:
this is a /dist/ import

Reply:
Import types is safe. During compilation this import is dropped since JS does not have types